Sources_v3/StreamChatUI/ChatChannel/ChatMessageReactionsView.swift

```diff
@@ -7,50 +7,50 @@ import UIKit
 
 // MARK: - Reaction
 
-class MessageReactionView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
-    var reaction: MessageReactionType? { didSet { updateContentIfNeeded() } }
-    var madeByCurrentUser: Bool = false { didSet { updateContentIfNeeded() } }
-    private let imageView: UIImageView = {
+open class ChatMessageReactionView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+    open var reaction: MessageReactionType? { didSet { updateContentIfNeeded() } }
+    open var madeByCurrentUser: Bool = false { didSet { updateContentIfNeeded() } }
+    public let imageView: UIImageView = {
```

Comment:
Yep, this is the exact reason why we have https://www.notion.so/GitHub-Etiquette-2cb296026db8404d88bc80bae809fd12#8d7acc904b8e4357983f28bab2e0d31b